package.json

```diff
@@ -49,6 +52,12 @@
     "dotenv": "^16.4.5",
     "helmet": "^7.1.0",
     "morgan": "^1.10.0",
+    "nodemailer": "^6.9.16",
+    "passport": "^0.7.0",
```

Comment:
Can you confirm that all these similar packages are needed? For example we have passport-google-oauth2 and 20 both. Not sure what's the difference.